python/srf/_pysrf/include/pysrf/node.hpp

```diff
@@ -173,7 +174,8 @@ namespace pysrf {
 
 template <typename InputT, typename ContextT = srf::runnable::Context>
 class PythonSink : public node::RxSink<InputT, ContextT>,
-                   public pysrf::AutoRegSinkAdapter<InputT>
+                   public pysrf::AutoRegSinkAdapter<InputT>,
+                   public pysrf::AutoRegEgressPort<InputT>
```

Comment:
Is there any reason why we wouldnt want to combine `AutoRegSinkAdapter` and `AutoRegEgressPort` into a single object? I cant think of a reason why you would want one but not the other

Reply:
Probably not; this gave us the most flexibility, but I suppose its not really an issue; I'll combine the inherited object.